src/NBitcoin.Tests/RPCClientTests.cs

```diff
@@ -100,6 +100,21 @@ public void CanGetBlockFromRPC()
             }
         }
 
+        [Fact]
+        public void TryValidateAddress()
+        {
+            using (var builder = NodeBuilder.Create())
+            {
+                var node = builder.CreateNode();
```

Comment:
108,111,113 - unvar please.

Reply:
@DanGould Just fix these and we are good to go 👍